jr-annotation-support/src/test/java/com/fasterxml/jackson/jr/annotationsupport/BasicAliasTest.java

```diff
@@ -34,6 +35,11 @@ public String getMiddle() {
         public void setLast(String str) { last = str; }
     }
 
+    record SnakeCaseRecord(
```

Comment:
Cannot be done here, as it requires JDK beyond 8. Any Record tests need to go in separate `jr-record-test` Maven sub-project.